pillar.example

```diff
@@ -1,5 +1,5 @@
 elasticsearch:
-  major_version: 2
+  version: 2.4.2-1
```

Comment:
Would unattended upgrades not break installed plugins, that tie into a specific version of ES? They usually break even on patch versions, IRC.